tests/test_autocomplete.py

```diff
@@ -186,7 +186,7 @@ def test_autocomplete_with_nlu_brand_no_focus(mock_autocomplete_get, mock_NLU_wi
     assert_ok_with(
         client,
         params={"q": "auchan", "lang": "fr", "limit": 7, "nlu": True},
-        expected_intention=[],
+        expected_intention=[{"filter": {"q": "auchan"}, "description": {"query": "auchan"},}],
```

Comment:
Nobody cares but ...
```suggestion
        expected_intention=[{"filter": {"q": "auchan"}, "description": {"query": "auchan"}}],
```
Y a une virugule en trop ... là ... :'D
(This is due to black's behaviour)

Reply:
fixed :D